src/arm/xnat.json

```diff
@@ -268,11 +269,11 @@
                     "dnsNameLabel": "[parameters('xnatCIName')]",
                     "ports": [
                         {
-                            "protocol": "tcp",
+                            "protocol": "TCP",
```

Comment:
Is changing the capitalization correct? my concern is that the system is capitalization-sensitive and this may not be recognized and hence triggering an error. If it not capitalization-sensitive then all good :)

Reply:
yes, this is defined in the JSON schema for Microsoft.ContainerInstance 
![image](https://user-images.githubusercontent.com/89279620/236270821-1f2e4dce-baad-4116-afa3-d981a09d6af8.png)

ref: https://schema.management.azure.com/schemas/2018-04-01/Microsoft.ContainerInstance.json

Reply:
Ship it! :)